src/engine/source/base/test/src/unit/utils/threadEventDispatcher_test.cpp

```diff
@@ -9,6 +9,7 @@
  * Foundation.
  */
 
+#include "base/logging.hpp"
```

Comment:
Given that we are modifying this file, let's re-order the headers based on the codding style proposed

Reply:
Done!